commercial/test/model/LookupTest.scala

```diff
@@ -29,7 +29,13 @@ import scala.concurrent.duration._
     Await.result(futureContents, timeout)
   }
 
-  "contentByShortUrls" should "find content for genuine URLs" in {
+  /*
+    Date: 02nd Dec 2020
+    Some tests (those with hardcoded short urls), come in two versions "old" and "new", this is due to a CAPI migration.
+    See (id: 288767d7-ba82-4d67-8fb3-9139e67b0f2e) , for details.
+   */
+
+  "contentByShortUrls (old version)" should "find content for genuine URLs" in {
```

Comment:
I think `new` and `old` are always harder to understand coming from the future. I think being explicit about their difference is more helpful:

- `old` &rarr; `gu.com`
- `new` &rarr; `www.theguardian.com`

```suggestion
  "contentByShortUrls (gu.com)" should "find content for genuine URLs" in {
```

It might also be worth checking for the naked root domain, which also seems to work, that is `https://theguardian.com/p/4vf6t`.

Not sure about how feasibly this is in Scala test, but you test against a string composed of every combination of the following:

```scala
val domains = Seq("gu.com", "guardian.com", "theguardian.com")
var shortUrls = Seq("p/3qeqm", "p/4v86p", "p/4vf6t")
```